handbook/documentation.rst

```diff
@@ -0,0 +1,33 @@
+####
+Developer Documentation
+####
+
+Overview
+********
+
+Documentation can live in lot's of places, we want to be clear about which documentation lives where most of the time so that finding docs is easier.  In general, keep the documents as close to the thing that they are describing as possible.
+
+Details
+****
+
+* READMEs, HOW-TOs, API Docs and ADRs should live with the code.
+
+  * API docs should be in swagger format.
+
+  * **Everything else should be RSTs**.  Yes, you already know markdown but RST is more powerful and robust and that is what we have chosen to use for consistency across the whole platform.  This_ is a good resources for learning RST.
+
+* Process docs such as scrum notes, retros, personal notes should not live in the repo and should generally be in confluence.
```

Comment:
Conversely, are we recommending that Confluence should only store process docs? It could be worth defining further what "process docs" mean actually, as I assume we don't include documentation that describe the processes themselves?